### PR TITLE
Allow access to all fbq events via raw query

### PR DESCRIPTION
**Pull request by Jamiewarb**
As it stands, there are elements of the FB Pixel that the wrapper prevents us accessing.

I've created a query method on the analytics.fbq object which will allow us to bypass the wrapper for things it doesn't yet handle.

This prevents us being limited by the wrapper. It is the same approach MatteoGabriele/vue-analytics takes.

An example use is `Vue.analytics.fbq.query('consent', 'revoke');`, as the wrapper currently does not support this.

https://developers.facebook.com/docs/facebook-pixel/events-advanced-use-cases/v3.1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
   },
   "main": "dist/vue-analytics-facebook-pixel.js",
   "repository": {
-    "url": "https://github.com/wemake-services/vue-analytics-facebook-pixel.git",
+    "url": "https://github.com/hex-digital/vue-analytics-facebook-pixel.git",
```
Comment (sobolevn) [inline package.json]: Can you please revert this change?

File: src/index.js [hunk 2]
```diff
@@ -62,6 +62,25 @@ const event = (name, data = {}) => {
...
+/**
+ * Submit a raw query to fbq, for when the wrapper limits user on what they need.
+ * This makes it still possible to access the plain Analytics api.
+ * @param  {String} appId
+ * @param  {object} [data={}]
+ */
+const query = (...args) => {
+  if (!_fbqEnabled()) return
+
+  if (config.debug) {
```
Comment (sobolevn) [inline src/index.js]: Maybe we should refactor `event()` to use `query()` underneath?

File: package.json (hunk 1 above)
Comment (Jamiewarb) [reply package.json]: Ah! Thank you. This was a mistake of me using my package temporarily in the mean time.

File: src/index.js (hunk 2 above)
Comment (Jamiewarb) [reply src/index.js]: Up to you! If you think that's a good pattern, then I can certainly make that change

**Comment by Jamiewarb** [comment]
@sobolevn I've updated the event and init methods to use query now, and removed that rogue URL. I force pushed over the top so it doesn't sit in this repo's history.

**Comment by sobolevn** [comment]
I will release `v2` on the weekends.
